Documentation/Updating.md

```diff
@@ -1,9 +1,22 @@
 # Updating the Microsoft Mixed Reality Toolkit
 
 - [2.1.0 to 2.2.0](#updating-210-to-220)
+- [2.2.0 to 2.3.0](#updating-220-to-230)
 - [2.0.0 to 2.1.0](#updating-200-to-210)
 - [RC2 to 2.0.0](#updating-rc2-to-200)
 
+## Updating 2.2.0 to 2.3.0
+
+- [API changes](#api-changes-in-230)
+
+### API changes in 2.3.0
+
+#### ScriptingUtilities.cs
+
+The ScriptingUtilities.cs file was moved from the MixedRealityToolkit\Utilities folder to MixedRealityToolkit\Utilities\Editor. As a result, the class has been moved to the Microsoft.MixedReality.Toolkit.Editor.Utilities assembly.
+
+In addition, the `AppendScriptingDefinitions` method has a new signature.  It no longer takes the fileName argument.
```

Comment:
I think that the purpose of this file is telling people prescriptively what to do, not just what happened - i.e. it's pretty obvious that it no longer takes a fileName argument, which folks would see when they get a compile break.

What would be useful here is a code snippet saying how to accomplish what they did before - i.e. if you passed a filename, you should do something like:

```c#
if (FileUtilities.CheckThatThingExistsYeah())
{
    AppendScriptingDefinitions(oh, yeah, wooooo);
}
```

And if you didn't pass a filename, it would just be:
```c#
    AppendScriptingDefinitions(oh, yeah, wooooo);
```

Reply:
Previous implementation restored and marked as obsolete.